### PR TITLE
enhancement: nsis script

### PR DESCRIPTION
**Pull request by jeeanribeiro**
## Summary
Enhances NSIS by changing from INetC to NScurl plugin and moves plugins/scripts to the correct build resources directory

## Changelog

```
- Adds NScurl DLLs
- Removes INetC DLLs
- Changes INetC request to NScurl request in custom NSIS script
- Adds DNS over HTTPS param with Cloudfare DNS query
- Adds cancel and resume NScurl params
- Moves files to the correct build resources directory
```

## Relevant Issues
- [x] #4529
- [x] #4530
- [ ] #4531 

### Platforms
- __Desktop__
  - [ ] MacOS
  - [ ] Linux
  - [ ] Windows
- __Mobile__
  - [ ] iOS
  - [ ] Android

## Checklist
- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/electron-builder-config.js [hunk 1]
```diff
@@ -116,6 +122,7 @@ const prereleaseNsisOptions = {
     nsis: {
         oneClick: false,
         allowToChangeInstallationDirectory: true,
+        debugLogging: true,
```
Comment (Tuditi) [inline packages/desktop/electron-builder-config.js:125]: I think this is not enough. You also need to add the following line to installer.nsh:

`LogSet on`

**Review by MarkNerdi — CHANGES_REQUESTED** [review]
Please rmove the .dll fils for inetc

**Review by Tuditi — CHANGES_REQUESTED** [review]
One thing that needs adding I think

**Review by Tuditi — APPROVED** [review]
LGTM! :fire:

**Pull request merged**